Tests/scripts/hook_validations/script.py

```diff
@@ -19,26 +19,28 @@ class ScriptValidator(object):
        old_script (dict): Json representation of the current script from master.
     """
 
-    def __init__(self, file_path, check_git=True, old_file_path=None):
+    def __init__(self, file_path, check_git=True, old_file_path=None, old_git_branch='master'):
         self.file_path = file_path
         self.current_script = {}
         self.old_script = {}
 
         if check_git:
             self.current_script = get_yaml(file_path)
+            old_git_link = '{root}/{branch}'.format(root=CONTENT_GITHUB_LINK, branch=old_git_branch)
             # The replace in the end is for Windows support
             if old_file_path:
-                git_hub_path = os.path.join(CONTENT_GITHUB_MASTER_LINK, old_file_path).replace("\\", "/")
+                git_hub_path = os.path.join(old_git_link, old_file_path).replace("\\", "/")
```

Comment:
remove `old_git_link`.

same for the `else`
```suggestion
                git_hub_path = os.path.join(CONTENT_GITHUB_LINK, old_git_branch, old_file_path).replace("\\", "/")
```